spec/std/http/cookie_spec.cr

```diff
@@ -252,6 +252,36 @@ module HTTP
       cookies["d"].value.should eq "e+f"
     end
 
+    it "allows retrieving the size of the cookies collection" do
+      cookies = Cookies.new
+      cookies.size.should eq 0
+      cookies << Cookie.new("1", "2")
+      cookies.size.should eq 1
+      cookies << Cookie.new("3", "4")
+      cookies.size.should eq 2
+    end
+
+    it "allows clearing the cookies collection" do
+      cookies = Cookies.new
+      cookies << Cookie.new("test_key", "test_value")
+      cookies << Cookie.new("a", "b")
+      cookies << Cookie.new("c", "d")
+      cookies.clear
+      cookies.size.should eq 0
+      cookies.empty?.should be_true
```

Comment:
size/empty expectations are a bit redundant.

You could prob just do `cookies.should be_empty`

Reply:
ah, didn't know about that helper

Reply:
Noted

Yeah my thought was this will fail if either `size` or `empty?` breaks but it's probably totally unnecessary.